simulation/__main__.py

```diff
@@ -84,7 +84,7 @@ def convert_str_to_intervention_list(intervention_string):
         "--all-agents-heat-pump-suitable",
         default=False,
         type=bool,
-        help="When True, 100%% of households are assumed suitable for heat pumps. When False, households are assigned a heat pump suitability as per the source data file.",
+        help="When True, 100% of households are assumed suitable for heat pumps. When False, households are assigned a heat pump suitability as per the source data file.",
```

Comment:
This isn't a typo. `argparse` lets you use [format specifiers](https://docs.python.org/3/library/argparse.html#help) so to get a `%` character.

Reply:
Yeah I realised that afterwards. This was since updated to `pc` though, which I think reads better in the codebase.